### PR TITLE
🚚  Add support for SPM and Apple Silicon

### PR DESCRIPTION
**Pull request by monchote**
- Adds support for SPM
- Adds support for Apple Silicon following [this method](https://www.notion.so/monzo/Adding-Apple-Silicon-support-to-an-existing-iOS-library-9da5384e07f14b219067b0ed75834a7f).

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// swift-tools-version:5.6
+import PackageDescription
+
+let package = Package(
+    name: "CardIO",
+    defaultLocalization: "en",
+    platforms: [
+        .iOS(.v13)
```
Comment (timmikeljmonzo) [inline Package.swift:8]: yes!

**Comment by timmikeljmonzo** [comment]
it looks great! Is there a Readme we could update as well @monchote ?

**Comment by monchote** [comment]
> it looks great! Is there a Readme we could update as well @monchote ?

@timmikeljmonzo I was planning to edit it with the built-in editor after merging, but I can also do it as part of this PR.

**Review by timmikeljmonzo — APPROVED** [review]
Nice work! ✅

**Pull request merged**